sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubClientBuilder.java

```diff
@@ -395,9 +395,8 @@ public EventProcessor buildEventProcessor() {
         EventPosition initialEventPosition =
             this.initialEventPosition == null ? EventPosition.earliest()
                 : this.initialEventPosition;
-
         return new EventProcessor(buildAsyncClient(), this.consumerGroupName,
```

Comment:
I don't think we have conventions around this, but `this.` is used inconsistently here to reference member variables. (ie. partitionManager doesn't have this). I'd prefer it without `this.` if there aren't any conflicting local variables (makes it more succinct), but I won't lose sleep if this change isn't in. 😆

Reply:
ditto; I'd personally limit `this` to use in getters/setters/ctors when required.